packages/terra-clinical-onset-picker/src/terra-dev-site/test/clinical-onset-picker/LessWeek.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import moment from 'moment';
+import OnsetPicker from '../../../OnsetPicker';
+
+class HandledOnsetExample extends React.Component {
```

Comment:
nit: class name can be changed to LessWeek or something more suitable for this test.